### PR TITLE
Fix/dont rename non cloudinary props

### PR DESCRIPTION
**Pull request by maoznir**

### CODE REVIEW

File: src/components/Video/Video.js [hunk 1]
```diff
@@ -70,11 +71,21 @@ class Video extends CloudinaryComponent {
     } = this.getMergedProps();
 
     options = CloudinaryComponent.normalizeOptions(options, {});
+    const ops = extractCloudinaryProps(options);
+    options = {...ops.cloudinaryProps, ...ops.cloudinaryReactProps};
+    /*
+    Object.keys(options).forEach(k=>{
```
Comment (patrick-tolosa) [inline src/components/Video/Video.js]: Do we need these comments?

Comment (maoznir) [reply src/components/Video/Video.js]: removed

File: src/components/Image/Image.js [hunk 2]
```diff
@@ -55,28 +55,25 @@ class Image extends CloudinaryComponent {
...
     // Set placeholder Id
     if (placeholder && attributes.id) {
       attributes.id = attributes.id + '-cld-placeholder';
     }
 
-    // Set dataSrc if lazy loading and not in view
+    // Set data-src if lazy loading and not in view
     if (!isInView && this.shouldLazyLoad(options)) {
-      attributes.dataSrc = attributes.dataSrc || attributes.src;
+      attributes['data-src'] = attributes.dataSrc || attributes.src;
```
Comment (asisayag2) [inline src/components/Image/Image.js:73]: @maoznir , why do we need this line AND lines 61-64?

File: src/components/Video/Video.js [hunk 3]
```diff
@@ -70,11 +71,16 @@ class Video extends CloudinaryComponent {
     } = this.getMergedProps();
 
     options = CloudinaryComponent.normalizeOptions(options, {});
+    const {cloudinaryProps, cloudinaryReactProps, nonCloudinaryProps} = extractCloudinaryProps(options);
+    options = {...cloudinaryProps, ...cloudinaryReactProps};
+
+    //const snakeCaseOptions = toSnakeCaseKeys(options);
```
Comment (asisayag2) [inline src/components/Video/Video.js:77]: Please remove

File: src/components/Video/Video.js [hunk 4]
```diff
@@ -70,11 +71,16 @@ class Video extends CloudinaryComponent {
...
     const snakeCaseOptions = Util.withSnakeCaseKeys(options);
     const cld = Cloudinary.new(snakeCaseOptions);
 
-    // Let cloudinary-core generate this video tag attributes
-    const tagAttributes = Util.withCamelCaseKeys(cld.videoTag(publicId, options).attributes());
+    // Use cloudinary-core to generate this video tag's attributes
+    let tagAttributes = cld.videoTag(publicId, options).attributes();
+    tagAttributes = {...Util.withCamelCaseKeys(tagAttributes), ...nonCloudinaryProps};
```
Comment (asisayag2) [inline src/components/Video/Video.js:83]: You have the cloudinary+react props in `options` and the non-cloudinary props in `nonCloudinaryProps`, why do we need to create a tag for getting those `tagAttributes`?

File: test/ImageTest.js [hunk 5]
```diff
@@ -94,6 +94,14 @@ describe('Image', () => {
 
     expect(tag.find('img').prop('src')).to.match(/fn_wasm:blur.wasm\/sample/);
   });
+  it('should not change kebab-case param names', () => {
```
Comment (asisayag2) [inline test/ImageTest.js:97]: it('should not change kebab-case for non-cloudinary param names', () => {

File: test/ImageTest.js [hunk 6]
```diff
@@ -94,6 +94,14 @@ describe('Image', () => {
...
+    let tag = mount(
+      <Image publicId="sample" cloudName="demo" data-testid="testing"/>
+    );
+
+    expect(tag.find('img').prop('data-testid')).to.equal('testing');
+    expect(tag.find('img').prop('datatestid')).to.equal(undefined);
```
Comment (asisayag2) [inline test/ImageTest.js:103]: `dataTestid`

File: test/VideoTest.js [hunk 7]
```diff
@@ -190,4 +190,13 @@ describe('Video', () => {
     expect(video.prop('plays_inline')).to.equal(undefined);
     expect(video.prop('auto_play')).to.equal(undefined);
   });
+  it('should not change kebab-case param names', () => {
```
Comment (asisayag2) [inline test/VideoTest.js:193]: it('should not change kebab-case for non-cloudinary param names', () => {

File: test/VideoTest.js [hunk 8]
```diff
@@ -190,4 +190,13 @@ describe('Video', () => {
...
+    let tag = mount(
+      <Video publicId="dog" cloudName="demo" data-testid="testing"/>
+    );
+
+    const video = tag.find('video');
+    expect(video.prop('data-testid')).to.equal("testing");
+    expect(video.prop('datatestid')).to.equal(undefined);
```
Comment (asisayag2) [inline test/VideoTest.js:200]: `dataTestid`

**Pull request merged**